[Update Account Settings] Header: icons, height, and font changes

kosmydel opened the pull request:
<!-- If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit. -->

### Details
<!-- Explanation of the change or anything fishy that is going on -->
This pull request adds icons to the `HeaderWithBackButton` component and uses it to add icons to screens in the Account Settings menu. It also adjusts header heights with new designs and applies new fonts.

### Fixed Issues
<!---
1. Please postfix `$` with a URL link to the GitHub issue this Pull Request is fixing. For example, `$ https://github.com/Expensify/App/issues/<issueID>`.
2. Please postfix  `PROPOSAL:` with a URL link to your GitHub comment, which contains the approved proposal (i.e. the proposal that was approved by Expensify).  For example, `PROPOSAL: https://github.com/Expensify/App/issues/<issueID>#issuecomment-1369752925`

Do NOT add the special GH keywords like `fixed` etc, we have our own process of managing the flow.
It MUST be an entire link to the github issue and your comment proposal ; otherwise, the linking and its automation will not work as expected.

Make sure this section looks similar to this (you can link multiple issues using the same formatting, just add a new line):

$ https://github.com/Expensify/App/issues/<issueID>
$ https://github.com/Expensify/App/issues/<issueID(comment)>

Do NOT only link the issue number like this: $ #<issueID>
--->
$ https://github.com/Expensify/App/issues/35609
$ https://github.com/Expensify/App/issues/35571
$ https://github.com/Expensify/App/issues/35632
PROPOSAL: N/A


### Tests
<!---
Add a numbered list of manual tests you performed that validates your changes work on all platforms, and that there are no regressions present.
Add any additional test steps if test steps are unique to a particular platform.
Manual test steps should be written so that your reviewer can repeat and verify one or more expected outcomes in the development environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->

- [ ] Verify that no errors appear in the JS console
- [ ] Go to the Account Settings.
- [ ] Verify icons in the central pane header on screens:
  - [ ] Profile
  - [ ] Wallet
  - [ ] Share code
  - [ ] Preferences
  - [ ] Security
  - [ ] General
  - [ ] About
- [ ] Verify header heights:
  - [ ] All headers should have 73px height except for ones in the Account Settings page.

### Offline tests
<!---
Add any relevant steps that validate your changes work as expected in a variety of network states e.g. "offline", "spotty connection", "slow internet", etc. Manual test steps should be written so that your reviewer and QA testers can repeat and verify one or more expected outcomes. If you are unsure how the behavior should work ask for advice in the `#expensify-open-source` Slack channel.
--->
Turn off network, verify the icons work.

### QA Steps
<!---
Add a numbered list of manual tests that can be performed by our QA engineers on the staging environment to validate that your changes work on all platforms, and that there are no regressions present.
Add any additional QA steps if test steps are unique to a particular platform.
Manual test steps should be written so that the QA engineer can repeat and verify one or more expected outcomes in the staging environment.

For example:
1. Click on the text input to bring it into focus
2. Upload an image via copy paste
3. Verify a modal appears displaying a preview of that image
--->

- [ ] Verify that no errors appear in the JS console
- [ ] Do the tests from above.

### PR Author Checklist
<!--
This is a checklist for PR authors. Please make sure to complete all tasks and check them off once you do, or else your PR will not be merged!
-->

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>
<img width="355" alt="Screenshot 2024-02-02 at 14 03 01" src="https://github.com/Expensify/App/assets/104823336/ca2eeafe-5422-4cc6-809e-1d5c3e60b7c2">



<!-- add screenshots or videos here -->
**Before font & height changes:**
https://github.com/Expensify/App/assets/104823336/cb156757-4de1-4bc0-89bb-00e0ab28fa7a


</details>

<details>
<summary>Android: mWeb Chrome</summary>
<img width="358" alt="Screenshot 2024-02-02 at 14 02 16" src="https://github.com/Expensify/App/assets/104823336/3f483cc5-68d9-4e87-bbfb-ec603f85f026">

<!-- add screenshots or videos here -->
**Before font & height changes:**
https://github.com/Expensify/App/assets/104823336/a8ba84df-743b-491d-8eed-b376a9486605


</details>

<details>
<summary>iOS: Native</summary>

![simulator_screenshot_53D7409A-2AC4-4A41-9F5B-87696B75C86F](https://github.com/Expensify/App/assets/104823336/2c39cd9f-9f64-4d22-a512-9318df6ddb10)

<!-- add screenshots or videos here -->
**Before font & height changes:**
https://github.com/Expensify/App/assets/104823336/1377c268-f3dd-4294-8090-aa60905c4dfd


</details>

<details>
<summary>iOS: mWeb Safari</summary>

![simulator_screenshot_DB63CC27-288A-4D82-A146-05318CFA1A40](https://github.com/Expensify/App/assets/104823336/9bde036f-2873-48c4-aed1-8a4a289008b0)



<!-- add screenshots or videos here -->
**Before font & height changes:**
https://github.com/Expensify/App/assets/104823336/8ec29a34-b4bb-471a-82bd-ca4da0aeab26


</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

![Screenshot 2024-02-02 at 13 18 18](https://github.com/Expensify/App/assets/104823336/ca74a67a-2054-4049-a120-87fa7c74bb0f)

<!-- add screenshots or videos here -->
**Before font & height changes:**
https://github.com/Expensify/App/assets/104823336/a20a6ae9-aa5e-4700-b6a1-eb93d711f5ee


</details>

<details>
<summary>MacOS: Desktop</summary>

<!-- add screenshots or videos here -->
**Before font & height changes:**
https://github.com/Expensify/App/assets/104823336/9f750591-467a-41c7-938f-e39c2da318c0


</details>

kosmydel committed:
add icons

kosmydel committed:
add remaining icons

kosmydel committed:
refactor

kosmydel committed:
Merge branch 'main' into update-account-settings/header-icons

kosmydel committed:
cleanup

shubham1206agra commented:
@mountiny Can you trigger an adhoc build to check the icons on physical devices?

kosmydel commented:
Converting to draft, as I will solve [this issue](https://github.com/orgs/Expensify/projects/94/views/2?pane=issue&itemId=51927558) in this PR as well. Going to reopen it in a moment.

kosmydel committed:
adjust headers heights

kosmydel committed:
fix height on small screens

kosmydel commented:
The PR is ready for review.

We are waiting for confirmation if the `80px` increased height should also be applied to the small screen or not. If yes, we just have to revert the last commit. So please keep this in mind when reviewing.

kosmydel committed:
Revert "fix height on small screens"

This reverts commit 02f327ed2e580f37978c5bfdd6a5704d92a753ca.

kosmydel committed:
Merge branch 'Expensify:main' into update-account-settings/header-icons

shawnborton commented:
Happy to help test this one too if we have more screenshots.

As you saw from my comment, my thinking is that:
- Any time a page header has an icon + h2 font style, it's always 80px, regardless of mobile or desktop
- If the page header does not have that fancy style (like page headers for the RHP, etc) we just use the style we previously had before the #ideal-nav changes

One thing that I am unsure of is what to do for page headers for chats on desktop. I _think_ our mockups had it so that on desktop, the chat header was 80px to match the height of the new switcher on the left. It looks something like this:
![CleanShot 2024-02-02 at 08 32 29@2x](https://github.com/Expensify/App/assets/2319350/7f5ff7bd-fc45-4d87-812b-dcac44cf0cfa)

This is so that we get a nice alignment going from the left nav area to the main content pane. However, I can see an argument that on mobile, we don't need the chat header to be a bit taller since you don't see the chat header next to the LHN - they are separate pages. So perhaps we only need to modify the chat header height on desktop? Thoughts on that @Expensify/design ?

shawnborton commented:
Super super down with that! I agree that it's a nice simplification.

kosmydel commented:
If we reach consensus, please update the Figma designs, and post and update with changes, so I can update the PR.

kosmydel commented:
Hey, can I ask for the final summary of the decisions? As in this thread, we are talking about 80px on desktop, while in [this issue](https://github.com/Expensify/App/issues/35571) we've decided to 72px. I'm a bit confused.

- What should be the central pane header height (mobile/desktop)?
- What should all other heights (mobile/desktop) be?

cc @shawnborton @dannymcclain

kosmydel committed:
fix colors in WS screen

kosmydel commented:
@hayata-suenaga I've fixed this issue in [this PR](https://github.com/Expensify/App/issues/35632).

 However, I've noticed that the colors also differ on the home screen (mWeb iOS). 
<img width="395" alt="Screenshot 2024-02-05 at 12 29 54" src="https://github.com/Expensify/App/assets/104823336/1c42e558-38cf-4281-b861-b3c8c1b0d64e">
But fixing this in the same way, as I've fixed the WS list screen, breaks the Android safe area:
<img width="785" alt="Screenshot 2024-02-05 at 12 26 28" src="https://github.com/Expensify/App/assets/104823336/53ad71fe-d40f-4389-9c01-a231a738edd0">

It has a different root cause. I wasn't able to fix it simply by configuration. It might be related to code in the `CustomStatusBarAndBackground` file (`updateStatusBarStyle` function). I think it should be treated as a separate issue.

kosmydel committed:
fix header heights

kosmydel committed:
Merge branch 'main' into update-account-settings/header-icons

kosmydel commented:
New design screenshots with paddings/dimensions (cc @shawnborton):

<details>
<summary>
Desktop
</summary>

![Screenshot 2024-02-05 at 16 09 32](https://github.com/Expensify/App/assets/104823336/9d61c20a-67d8-49aa-a339-747a37f54adf)

![Screenshot 2024-02-05 at 16 09 41](https://github.com/Expensify/App/assets/104823336/93a7c39c-2e52-434b-a30b-6e56e35d6150)

![Screenshot 2024-02-05 at 16 10 59](https://github.com/Expensify/App/assets/104823336/83374b6c-5830-41c9-8c56-d50e5ec7f47e)

![Screenshot 2024-02-05 at 16 11 08](https://github.com/Expensify/App/assets/104823336/8b21e65a-bf47-4f5f-959c-d5d93f3cc8ae)

![Screenshot 2024-02-05 at 16 11 39](https://github.com/Expensify/App/assets/104823336/1f802f57-67d2-410d-8627-6a3681514173)

</details>

<details>
<summary>
Mobile
</summary>

![Screenshot 2024-02-05 at 16 13 40](https://github.com/Expensify/App/assets/104823336/28d621c9-6e8d-434d-9a7b-14045f6c16f9)

![Screenshot 2024-02-05 at 16 13 51](https://github.com/Expensify/App/assets/104823336/3c524596-5ebc-469e-97d0-26d03cb74a4b)

![Screenshot 2024-02-05 at 16 14 00](https://github.com/Expensify/App/assets/104823336/8206cb4a-d6aa-4a81-9d27-73621c6b6660)

![Screenshot 2024-02-05 at 16 14 10](https://github.com/Expensify/App/assets/104823336/5bd9e94b-edae-45cc-a369-e62273511108)


</details>

--- 

> @kosmydel something that our designer @dubielzyk-expensify pointed out was that there is a pretty big gap on iOS below the dynamic island and the LHN header content. Is it possible to reduce that space by a small amount so we don't eat up as much non-scrollable space on the view?

I tried to simply change the top padding to 0px on mobiles (currently it uses a top padding of 20px), but it broke Android. Something is wrong with the SafeAreaInsets on Android devices that would require deeper investigation. It might be also related to [this bug](https://github.com/Expensify/App/pull/35564#issuecomment-1926772435) mentioned above. 

<img width="764" alt="Screenshot 2024-02-05 at 16 07 17" src="https://github.com/Expensify/App/assets/104823336/f43381b9-a752-4fc4-a058-103615ba41cd">

So it would be better, to treat it as a separate issue.

kosmydel committed:
cleanup

shawnborton commented:
For this screen here:
![image](https://github.com/Expensify/App/assets/2319350/646af69f-274b-418d-9393-db088ff44ec5)

So we maintain our base4 mentality, can we make the area that wraps the logo/workspace name + page name with a fixed height of say 24px? And then just vertically center everything in there.

shawnborton commented:
Ah sorry if this wasn't clear, but I think the idea is that any time the page content header has an illustration + h2 text, it always is at 80px tall. So on mobile, this should actually be 80px and not 72px:
![image](https://github.com/Expensify/App/assets/2319350/7f6deb7e-a3ce-45fe-aa8a-8e58b6502ed9)

shawnborton commented:
For the page header views, it seems weird to only give the right side padding:
![image](https://github.com/Expensify/App/assets/2319350/06f0dd7c-0766-4895-81ee-e2c767a275d4)

Any idea why we are setting those up that way? Why wouldn't both sides have the padding, like so?
![CleanShot 2024-02-05 at 11 09 18@2x](https://github.com/Expensify/App/assets/2319350/79c4eaf3-78ac-4ff8-a50f-7f0b30023497)

hayata-suenaga commented:
> I think it should be treated as a separate issue.

thank you for confirmation. I'll keep both issues then 👍

kosmydel committed:
set breadcrumbs height

kosmydel commented:
> So we maintain our base4 mentality, can we make the area that wraps the logo/workspace name + page name with a fixed height of say 24px? And then just vertically center everything in there.

I've changed it to a fixed 24px.

> h sorry if this wasn't clear, but I think the idea is that any time the page content header has an illustration + h2 text, it always is at 80px tall. So on mobile, this should actually be 80px and not 72px:

Done.

> Any idea why we are setting those up that way? Why wouldn't both sides have the padding, like so?

I'm not sure, but it might be related to handling the back button on small screens.

kosmydel committed:
increase height on illustration header

kosmydel committed:
Merge branch 'main' into update-account-settings/header-icons

kosmydel committed:
Merge branch 'Expensify:main' into update-account-settings/header-icons

kosmydel commented:
@shubham1206agra it is ready for review.

shawnborton commented:
Going to do a test build of this one so we can take it for a spin as well

mountiny commented:
Running the build

kosmydel commented:
> Can you confirm that there is just 12px of padding under the page header, and then the content begins?

I think it's 16px. But this is going to be redesigned in [this PR](https://github.com/Expensify/App/pull/35662) anyway.

shawnborton commented:
Cool, initial testing on that build seems to look good to me 👍

shubham1206agra commented:
The profile Illustration looks oddly misaligned slightly. Can you please check the SVG to align it maybe?

dannymcclain commented:
> The profile Illustration looks oddly misaligned slightly. Can you please check the SVG to align it maybe?

@shubham1206agra can you explain a bit more? If you're referring to what I'm showing in my screenshots here, it is to be expected and I don't think it's an issue. It stems from all our illustrations using a fixed size of 48x48px and having to optically resize each illustration inside those bounds to make them visually feel about the same size.

You'll notice a similar thing happening with the lock illustration on the security page because that illustration is much taller than it is wide.

If you're talking about something else—please let me know!

![CleanShot 2024-02-06 at 09 53 04@2x](https://github.com/Expensify/App/assets/7516624/ce45f11d-08ed-4d39-9a13-eabf039e6fec)

shubham1206agra commented:
> The Profile icon is not aligned vertically on Figma. It might be this.
> 
> ![Screenshot 2024-02-06 at 16 46 17](https://private-user-images.githubusercontent.com/104823336/302693771-d1a3fde5-59b1-41ce-8198-76730634fd49.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcyMzU0MTYsIm5iZiI6MTcwNzIzNTExNiwicGF0aCI6Ii8xMDQ4MjMzMzYvMzAyNjkzNzcxLWQxYTNmZGU1LTU5YjEtNDFjZS04MTk4LTc2NzMwNjM0ZmQ0OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIwNlQxNTU4MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yMzMxYmUxYTMwY2M4NWJiNTkyOWZjMTY5MWEwMjQzYzhmZmI0M2NjOTk0Y2Y4ZmQwNjNhZWFmOGU5MjM3ODg2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.oaeA9iNdlxylWUaRsYgPhyWLRFABqWgbslekSSXu8n8)

Yes. I am talking about this only.

shubham1206agra commented:
See alignment with back button
![image](https://github.com/Expensify/App/assets/58412969/4fc45611-bcc3-4153-bf0f-d2cb8c416e2d)

kosmydel commented:
I can align it in InkScape, but it would be better to fix it on Figma as well.
Wrapping paths in a group and applying this transform fixes it: ```<g id="g4" transform="translate(-1e-4,1.41179)">```

shawnborton commented:
Can you inspect and show us how the views are all lining up? As long as everything is vertically centered, I think we're fine. I don't think we need to modify the svgs...

dannymcclain commented:
Ahh gotcha! Thanks for pointing that out. I just updated it in Figma but I'll attach the SVG here as well. 
[simple-illustration__profile.svg.zip](https://github.com/Expensify/App/files/14183009/simple-illustration__profile.svg.zip)

@shawnborton We did actually need to edit the SVG—it was not quite right and I think this is better now.

kosmydel committed:
fix Profile icon

shawnborton commented:
Can you show me a before/after? I'm not disagreeing, but I don't think I am seeing it so just want to make sure I can spot the misalignment.

kosmydel commented:
Before:
<img width="144" alt="Screenshot 2024-02-06 at 17 06 36" src="https://github.com/Expensify/App/assets/104823336/8f923004-c7fa-4f68-9888-eeb493eea286">


After:
<img width="151" alt="Screenshot 2024-02-06 at 17 06 09" src="https://github.com/Expensify/App/assets/104823336/05ad0683-9663-44b7-995a-17853f4c0c9b">

kosmydel commented:
> Ahh gotcha! Thanks for pointing that out. I just updated it in Figma but I'll attach the SVG here as well. [simple-illustration__profile.svg.zip](https://github.com/Expensify/App/files/14183009/simple-illustration__profile.svg.zip)
> 
> @shawnborton We did actually need to edit the SVG—it was not quite right and I think this is better now.

I've downloaded it from Figma, as it was updated (I guess)?

kosmydel commented:
Cc @shubham1206agra ready!

shubham1206agra commented:
Looks fine to me now. I will post the checklist in the morning.

shubham1206agra commented:
<img width="1512" alt="Screenshot 2024-02-07 at 8 25 14 PM" src="https://github.com/Expensify/App/assets/58412969/fe3513da-2dba-41ef-9d43-af529b2b0fa2">

Can you confirm the height here?

shawnborton commented:
I think all of those are going to be changed with the upcoming account settings project. I think @kosmydel might be able to confirm that.

shubham1206agra commented:
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

<img width="438" alt="Image" src="https://github.com/Expensify/App/assets/58412969/4fc45611-bcc3-4153-bf0f-d2cb8c416e2d">

</details>

<details>
<summary>Android: mWeb Chrome</summary>

<img width="438" alt="Screenshot 2024-02-07 at 8 32 35 PM" src="https://github.com/Expensify/App/assets/58412969/4aeed86d-c8f1-4aa4-ab36-7499c604c37e">

</details>

<details>
<summary>iOS: Native</summary>

<img width="413" alt="Screenshot 2024-02-07 at 8 41 17 PM" src="https://github.com/Expensify/App/assets/58412969/62dbc635-0cf8-4344-9a21-e408467eaff7">

</details>

<details>
<summary>iOS: mWeb Safari</summary>

<img width="404" alt="Screenshot 2024-02-07 at 8 30 01 PM" src="https://github.com/Expensify/App/assets/58412969/c89c95fe-0b70-489f-9044-e37830c315c4">

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

<img width="1512" alt="Screenshot 2024-02-07 at 8 27 45 PM" src="https://github.com/Expensify/App/assets/58412969/005cf58d-753b-403b-95b1-7da41b882bac">

</details>

<details>
<summary>MacOS: Desktop</summary>

<img width="1512" alt="Screenshot 2024-02-07 at 8 35 08 PM" src="https://github.com/Expensify/App/assets/58412969/8000ac2b-08a1-43b7-99f3-6f9e691dc41d">

</details>

kosmydel commented:
> I think all of those are going to be changed with the upcoming account settings and simplified collect project. I think @kosmydel might be able to confirm that.

I haven't changed the Workspace Settings at all, as there are no illustrations. This is probably a part of future project.

mountiny commented on src/components/Icon/Illustrations.ts:
```diff
@@ -50,6 +52,9 @@ import MoneyBadge from '@assets/images/simple-illustrations/simple-illustration_
 import OpenSafe from '@assets/images/simple-illustrations/simple-illustration__opensafe.svg';
+import PalmTree from '@assets/images/simple-illustrations/simple-illustration__palmtree.svg';
+import Profile from '@assets/images/simple-illustrations/simple-illustration__profile.svg';
+import QrCode from '@assets/images/simple-illustrations/simple-illustration__qr-code.svg';
```
Naming styling, can be fixed in a follow up
```suggestion
import QRCode from '@assets/images/simple-illustrations/simple-illustration__qr-code.svg';
```

kosmydel replied:
Going to rename it [here](https://github.com/Expensify/App/pull/35672).

mountiny reviewed (approved):
Thanks for the changes, lets update the name variable in a follow up

OSBotify commented:
:hand: This PR was not deployed to staging yet because QA is ongoing. It will be automatically deployed to staging after the next production release.

kosmydel commented:
There is one regression comming from this PR: missing background color in workspace empty state.

It's going to be fixed in [this PR](https://github.com/Expensify/App/pull/35672).

![simulator_screenshot_F3A32940-09BD-466C-B7D6-85CC5B3C18E1](https://github.com/Expensify/App/assets/104823336/8edf4113-dedb-4b41-83b7-cfe82c9d2a81)


cc @mountiny, @shubham1206agra

OSBotify commented:
🚀 [Deployed](https://github.com/Expensify/App/actions/runs/7834036228) to staging by https://github.com/mountiny in version: 1.4.39-0 🚀

platform | result
---|---
🤖 android 🤖|success ✅
🖥 desktop 🖥|success ✅
🍎 iOS 🍎|success ✅
🕸 web 🕸|success ✅
